.storybook/setupTests.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import Enzyme, {shallow, render, mount} from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import MockDate from 'mockdate';
```

Comment:
This is what we are using in the widgets repo. Works really well.

Reply:
I tried using it before when I first opened my PR, and I couldn't seem to make it work. I realized recently it was because `date-fns` was being mocked, and also the `Date` class in this file 🤦🏻‍♀️